src/qt/transactionview.cpp

```diff
@@ -415,6 +415,9 @@ void TransactionView::bumpFee()
 
     // Bump tx fee over the walletModel
     model->bumpFee(hash);
+
+    // Update the table
+    model->getTransactionTableModel()->updateTransaction(hashQStr, CT_UPDATED, false);
```

Comment:
Heh. Yes. This is really surprising and yes, I think we should remove it (in a clean follow up PR).